components/FormCreate.js

```diff
@@ -0,0 +1,145 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: #ffffff;
+  background-color: #3e407d;
+  border: 5px solid #3e407d;
+  border-radius: 5px;
+  margin-left: 10px;
+  font-size: 17px;
+`;
+
+const StyledForm = styled.form`
+  margin: 10px;
+  padding-top: 5px;
+  display: flex;
+  flex-direction: column;
+  gap: 1.1rem;
+`;
+
+const StyledSection = styled.section`
+  display: flex;
+  flex-direction: column;
+  gap: 0.6rem;
+`;
+
+const StyledInput = styled.input`
+  padding: 0.5rem;
+  border: 2px solid black;
+  border-radius: 0.5rem;
+`;
+
+const StyledSelect = styled.select`
+  padding: 0.5rem;
+  border: 2px solid black;
+  border-radius: 0.5rem;
+`;
+
+const StyledTextarea = styled.textarea`
+  padding: 0.5rem;
+  border: 2px solid black;
+  border-radius: 0.5rem;
+`;
+
+const StyledButton = styled.button`
+  position: center;
+  color: #ffffff;
+  background-color: #4caf50;
+  border: 7px solid #4caf50;
+  border-radius: 5px;
+  font-size: 18px;
+`;
+
+export default function FormCreate({ addActivity }) {
+  const router = useRouter();
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const form = event.target;
+    const formData = new FormData(form);
+    const data = Object.fromEntries(formData);
```

Comment:
You could destructure data and you do not need to type "data.xzy" several times in the const newActivity